conf/solr/crawldb/conf/managed-schema

```diff
@@ -114,25 +114,32 @@
    <field name="text" type="text_general" stored="false" indexed="true" multiValued="true"/>
 
    <!-- Crawl DB Static Fields -->
-   <field name="id" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" /> 
+   <field name="id" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" />
    <field name="url" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" />
    <field name="jobId" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" />
-   <field name="group" type="string" indexed="true" stored="true" required="true"  multiValued="false" docValues="true" /> 
-   <field name="lastFetchedAt" type="date" indexed="true" stored="true" multiValued="false" docValues="true" />
+    <field name="group" type="string" indexed="true" stored="true" required="true"  multiValued="false" docValues="true" />
+    <field name="group_id" type="string" indexed="true" stored="true" required="true"  multiValued="false" docValues="true" />
```

Comment:
Group is a very good information. For the query, grouping by "group" leads to a non equiprobable distribution in term of size of groups. So I added a new field in which I populate a number which is equally distributed. I used a random generator to fill that "group_id".